.coveragerc

```diff
@@ -0,0 +1,6 @@
+[report]
```

Comment:
In all of the other python projects in Ocado, we use something like:
```
$ cat setup.cfg
[coverage:run]
source = django_autoconfig
```

Can we move this to the standard coverage config in setup.cfg?